chapters/arrays.tex

```diff
@@ -1417,12 +1417,12 @@ \section{Empty Arrays}\label{empty-arrays}
 
 Empty matrices can be constructed with the fill function.  For example:
 \begin{lstlisting}[language=modelica]
-Real A[:,:] = fill(0.0, 0, 1); // a Real 0 x 1 matrix
```

Comment:
I actually prefer the original here, as adding _empty_ to me suggests that there could be _non-empty_ 0 x 1 matrices.
Since it is in the section about Empty Arrays I don't think we need to be that explicit that it is empty.

Reply:
But shouldn't we remove "empty" from all comments then?

Reply:
Not the ones that say "empty matrices" (of different dimensions) and "empty vector" that don't specify the size.
What I don't like is the redundancy of writing empty _and_ writing an empty size.

Reply:
OK.